### ISSUE TITLE
SSL Issues with pipenv (but not pip). Out of nowhere. (code 590, TLS version issues)

### ISSUE THREAD

**Issue by abrudnick** [pypa/pipenv#2008]
Be sure to check the existing issues (both open and closed!).

Describe the issue briefly here.

Please run `$ python -m pipenv.help`, and paste the results here.
<details><summary>$ python -m pipenv.help output</summary>

Pipenv version: `'11.10.0'`

Pipenv location: `'/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/site-packages/pipenv'`

Python location: `'/usr/local/bin/python3'`

Other Python installations in `PATH`:

  - `2.6`: `/usr/bin/python2.6`
  - `2.6`: `/usr/bin/python2.6`
  - `2.7`: `/usr/bin/python2.7`
  - `2.7`: `/usr/bin/python2.7`
  - `3.6`: `/usr/local/bin/python3.6m`
  - `3.6`: `/usr/local/bin/python3.6`
  - `3.6`: `/Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6`
  - `3.6`: `/usr/local/bin/python3.6`
  - `3.6`: `/Users/###############/anaconda3/bin/python3.6`

  - `2.7.10`: `/usr/bin/python`
  - `3.6.2`: `/Users/############/anaconda3/bin/python`
  - `3.6.1`: `/usr/local/bin/python3`
  - `3.6.1`: `/Library/Frameworks/Python.framework/Versions/3.6/bin/python3`
  - `3.6.1`: `/usr/local/bin/python3`
  - `3.6.2`: `/Users/##########/anaconda3/bin/python3`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.6.1',
 'os_name': 'posix',
 'platform_machine': 'x86_64',
 'platform_python_implementation': 'CPython',
 'platform_release': '16.7.0',
 'platform_system': 'Darwin',
 'platform_version': 'Darwin Kernel Version 16.7.0: Thu Jan 11 22:59:40 PST '
                     '2018; root:xnu-3789.73.8~1/RELEASE_X86_64',
 'python_full_version': '3.6.1',
 'python_version': '3.6',
 'sys_platform': 'darwin'}
```

System environment variables:

  - `TERM_PROGRAM`
  - `TERM`
  - `SHELL`
  - `TMPDIR`
  - `Apple_PubSub_Socket_Render`
  - `TERM_PROGRAM_VERSION`
  - `TERM_SESSION_ID`
  - `USER`
  - `SSH_AUTH_SOCK`
  - `__CF_USER_TEXT_ENCODING`
  - `PATH`
  - `PWD`
  - `LANG`
  - `XPC_FLAGS`
  - `XPC_SERVICE_NAME`
  - `HOME`
  - `SHLVL`
  - `LOGNAME`
  - `_`
  - `OLDPWD`
  - `PYTHONDONTWRITEBYTECODE`
  - `PIP_PYTHON_PATH`
  - `PYTHONUNBUFFERED`

Pipenv–specific environment variables:

Debug–specific environment variables:

  - `PATH`: `/usr/local/bin:/Library/Frameworks/Python.framework/Versions/3.6/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/TeX/texbin:/Users/########/anaconda3/bin`
  - `SHELL`: `/bin/bash`
  - `LANG`: `en_US.UTF-8`
  - `PWD`: `/Users/###############/Django/plaidquickstart/quickstart/python`

---------------------------

Contents of `Pipfile` ('/Users/##########/Django/plaidquickstart/quickstart/python/Pipfile'):

```toml
[[source]]
url = "https://pypi.python.org/simple"
verify_ssl = true
name = "pypi"

[packages]
Flask = "==0.12"
"Jinja2" = "==2.9.5"
MarkupSafe = "==0.23"
Werkzeug = "==0.11.15"
click = "==6.7"
itsdangerous = "==0.24"
requests = "==2.13.0"
wsgiref = "==0.1.2"
plaid-python = "<2.1,>=2.0"

[dev-packages]

```

Contents of `Pipfile.lock` ('/Users/##########/Django/plaidquickstart/quickstart/python/Pipfile.lock'):

```json
{
    "_meta": {
        "hash": {
            "sha256": "732faeb18a1d9ade020610dc8559d4dd5d5927ad511386c8df965e3948363c1b"
        },
        "host-environment-markers": {
            "implementation_name": "cpython",
            "implementation_version": "0",
            "os_name": "posix",
            "platform_machine": "x86_64",
            "platform_python_implementation": "CPython",
            "platform_release": "16.7.0",
            "platform_system": "Darwin",
            "platform_version": "Darwin Kernel Version 16.7.0: Thu Jan 11 22:59:40 PST 2018; root:xnu-3789.73.8~1/RELEASE_X86_64",
            "python_full_version": "2.7.10",
            "python_version": "2.7",
            "sys_platform": "darwin"
        },
        "pipfile-spec": 6,
        "requires": {},
        "sources": [
            {
                "name": "pypi",
                "url": "https://pypi.python.org/simple",
                "verify_ssl": true
            }
        ]
    },
    ###Some details hidden because they're nonmaterial (hashes, etc).###
</details>

If you're on MacOS, just run the following:

    $ python -m pipenv.help | pbcopy

------------





##### Expected result

I expected pipenv to work -- pip had the same SSL issue, and I upgraded it using curl and it worked fine. Just upgraded pipenv to 11 (as shown above) and there's still an issue with installing packages.

##### Actual result

When possible, provide the verbose output (`--verbose`), especially for locking and dependencies resolving issues.


Collecting click==6.7 ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/8 — 00:00:00
  Could not fetch URL https://pypi.python.org/simple/click/: There was a problem confirming the ssl certificate: [SSL: TLSV1_ALERT_PROTOCOL_VERSION] tlsv1 alert protocol version (_ssl.c:590) - skipping

  Could not find a version that satisfies the requirement click==6.7 (from -r /var/folders/94/xgvg2c1n6sg91176m_rdx4z00000gn/T/pipenv-3u747fgx-requirements/pipenv-k_ubf78t-requirement.txt (line 1)) (from versions: )
No matching distribution found for click==6.7 (from -r /var/folders/94/xgvg2c1n6sg91176m_rdx4z00000gn/T/pipenv-3u747fgx-requirements/pipenv-k_ubf78t-requirement.txt (line 1))
##### Steps to replicate

Provide the steps to replicate (which usually at least includes the commands and the Pipfile).

Basically, the SSL check is failing. I'd rather not get rid of the SSL checking (though I know that's a way around this). The issue just came up today. Thoughts?

**Comment by abrudnick** [comment]
Not sure what happened but just resolved itself. Must've been an issue on the other side.
